contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -4,18 +4,51 @@ pragma solidity >=0.8.7 <0.9.0;
 import '../../interfaces/IDCAStrategies.sol';
 
 abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHandler {
-  /// @inheritdoc IDCAStrategiesManagementHandler
-  function getStrategy(uint80 _strategyId) external view override returns (IDCAStrategies.Strategy memory) {}
+  struct StrategyOwnerAndVersion {
+    address owner;
+    uint96 latestVersion;
+  }
+
+  mapping(uint80 => StrategyOwnerAndVersion) internal _strategies;
+  mapping(string => uint80) public getStrategyIdByName;
+  mapping(uint80 => string) public getStrategyNameById;
```

Comment:
I think`getStrategyNameById` can be internal, since we will already expose it under `getStrategy`